mobius-coroutines/src/main/java/com/spotify/mobius/coroutines/CoroutinesSubtypeEffectHandlerBuilder.kt

```diff
@@ -23,68 +30,152 @@ import kotlin.reflect.KClass
  * instances of the builder are mutable and not thread-safe.
  */
 class CoroutinesSubtypeEffectHandlerBuilder<F : Any, E : Any> {
-    private val effectHandlersMap = mutableMapOf<KClass<out F>, suspend (F) -> Flow<E>>()
+    private val effectHandlersMap = mutableMapOf<KClass<out F>, EffectHandler<F, E>>()
 
     inline fun <reified G : F> addAction(
+        cancellationPolicy: CancellationPolicy = RunSequentially,
         crossinline action: suspend () -> Unit
-    ) = addFlowProducer<G> {
+    ) = addEffectHandler<G>(cancellationPolicy) { _, _ ->
         action.invoke()
-        flowOf()
     }
 
     inline fun <reified G : F> addConsumer(
+        cancellationPolicy: CancellationPolicy = RunSequentially,
```

Comment:
I thought that RunConcurrently would be the the non-breaking change? But I suppose sequential execution probably makes more sense as a default

Reply:
Previously there were no guaranties, I picked this one as the "safe one" since it does not involves concurrency or cancellations (for the same type of event).